### ISSUE TITLE
Bug in TemporalEvenCrop

### ISSUE THREAD

**Issue by ombretta** [kenshohara/3D-ResNets-PyTorch#254]
There is a bug in temporal_transforms.py, line 115. 
`sample = list(range(begin_index, end_index))` 
overwrites the subsampling from main.py, line 229-230. Because of that, the videos from the validation set are always sampled with stride 1, even if opt.sample_t_stride > 1.
Can be fixed by replacing line 115 in temporal_transforms.py with 
`sample = list(frame_indices[begin_index:end_index])`
